File: gaia-sdk-python/gaia_sdk/http/request/InitBinaryWriteImpulse.py
class InitBinaryWriteImpulse:
    uri: str
    override: bool

    def __init__(self, uri: str, override: bool):
        self.uri = uri
        self.override = override

    def __eq__(self, other):
        return self.uri == other.uri \
               and self.override == other.override

    def __repr__(self):
        return {'uri': self.uri,
                'override': self.override}

File: gaia-sdk-java/gaia-sdk-api/src/main/kotlin/gaia/sdk/api/data/request/InitBinaryWriteImpulse.kt
package gaia.sdk.api.data.request

import com.fasterxml.jackson.annotation.JsonCreator
import com.fasterxml.jackson.annotation.JsonProperty

data class InitBinaryWriteImpulse @JsonCreator constructor(
        @JsonProperty("uri") val uri: String,
        @JsonProperty("override") val override: Boolean)
